frontend/src/scenes/performance/WebPerformance.tsx

```diff
@@ -322,16 +382,14 @@ const EventsWithPerformanceTable = (): JSX.Element => {
 
 const DebugPerfData = (): JSX.Element => {
     const { currentEvent } = useValues(webPerformanceLogic)
-    return (
+    return currentEvent ? (
         <Collapse>
             <Collapse.Panel header="Performance Debug Information" key="1">
-                <pre>
-                    {currentEvent
-                        ? JSON.stringify(JSON.parse(currentEvent.properties.$performance_raw), undefined, 2)
-                        : null}
-                </pre>
+                <pre>{JSON.stringify(JSON.parse(currentEvent.properties.$performance_raw), undefined, 2)}</pre>
             </Collapse.Panel>
         </Collapse>
+    ) : (
+        <></>
```

Comment:
This is a nit, but a `null` is much better, more explicit at conveying the emptiness of the result than an empty fragment (which for all the programmer – or React – knows, can contain just about anything).